mlflow/pipelines/pipeline.py

```diff
@@ -81,12 +81,12 @@ def run(self, step: str = None) -> None:
         # TODO Record performance here.
         # Always resolve the steps to load latest step modules before execution.
         self._steps = self._resolve_pipeline_steps()
```

Comment:
Is this line necessary now? Looks to be unused.

Reply:
It is needed in subclass https://github.com/mlflow/mlflow/blob/ff874bc935a923c4eafa9ff091145fa0492c8c13/mlflow/pipelines/regression/v1/pipeline.py#L453